graph-db/indexer/handler.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports.index = async event => {
```

Comment:
This function will be completely replaced with the actual indexing code that is coming when we work on CMR-7307. So ignore what's happening inside here.